dynamics-nav-app/LocalFunctionality/Belgium/how-to-set-up-declaration-types.md

```diff
@@ -23,16 +23,16 @@ In [!INCLUDE[navnow](../../includes/navnow_md.md)], there are two types of decla
 
 The type of declaration depends on the amount of shipped or received goods. To determine the type of declaration that you should use, see the [National Bank of Belgium](https://go.microsoft.com/fwlink/?LinkId=163064) web site.  
 
-When using the extended declaration, you will also need to set up an Incoterm in Intrastat Declaration for each shipping method. For more information, see [Incoterm in Intrastat Decl. Field](ms-its:be_lf_a.chm::/T_10_11300.htm).  
+When using the extended declaration, you will also need to set up an Incoterm in Intrastat Declaration for each shipping method.
 
 ## To set up declaration types  
 
 1.  Choose the ![Search for Page or Report](../../media/ui-search/search_small.png "Search for Page or Report icon") icon, enter **General Ledger Setup**, and then choose the related link.  
 2.  On the **General** FastTab, select the **Simplified Intrastat Decl.** check box to set up a simplified declaration type. Clear this field to use extended declaration.  
 3.  Choose the **OK** button.  
-
+microsoft.com
```

Comment:
Why was this added? Could be removed.

Reply:
Don't know how that got there, but I took it out as suggested. Thanks.